--- origsrc/pango-1.34.1/configure.ac	2013-05-13 10:31:52.000000000 -0500
+++ src/pango-1.34.1/configure.ac	2013-11-18 13:45:39.746513100 -0600
@@ -145,17 +145,6 @@ AC_CHECK_HEADERS(unistd.h sys/mman.h)
 AC_LIBTOOL_WIN32_DLL
 AM_DISABLE_STATIC
 
-if test "$pango_os_win32" = "yes"; then
-  if test x$enable_static = xyes -o x$enable_static = x; then
-    AC_MSG_WARN([Disabling static library build, must build as DLL on Windows.])
-    enable_static=no
-  fi
-  if test x$enable_shared = xno; then
-    AC_MSG_WARN([Enabling shared library build, must build as DLL on Windows.])
-  fi
-  enable_shared=yes
-fi
-
 AM_PROG_LIBTOOL
 dnl when using libtool 2.x create libtool early, because it's used in configure
 m4_ifdef([LT_OUTPUT], [LT_OUTPUT])
